.github/actions/spelling/allow.txt

```diff
@@ -0,0 +1,53 @@
+actionsearch
+appmenu
+appmenumodel
+appmenuwidget
+batterywidget
+blurwindow
+CMake
+cmake
+controlwidget
+cpp
+datetime
+datetimewidget
+dbusmenu
+dbusmenuimporter
+dbusmenushortcut
+dbusmenutypes
+dbustypes
+extensionwidget
+github
+https
+kde
```

Comment:
note that unlike expect, an entry here doesn't cover other case variants (so, `KDE` will likely be in `expect.txt` -- it could be moved here).